geode-redis/src/main/java/org/apache/geode/redis/internal/netty/ExecutionHandlerContext.java

```diff
@@ -247,25 +306,16 @@ private RedisResponse handleUnAuthenticatedCommand(Command command) {
     return response;
   }
 
-  public EventLoopGroup getSubscriberGroup() {
-    return subscriberGroup;
-  }
-
-  public void changeChannelEventLoopGroup(EventLoopGroup newGroup) {
-    if (newGroup.equals(channel.eventLoop())) {
-      // already registered with newGroup
-      return;
-    }
-    channel.deregister().addListener((ChannelFutureListener) future -> {
-      newGroup.register(channel).sync();
-    });
-  }
-
-  private void logResponse(RedisResponse response) {
+  private void logResponse(RedisResponse response, String extraMessage, Throwable cause) {
     if (logger.isDebugEnabled() && response != null) {
       ByteBuf buf = response.encode(new UnpooledByteBufAllocator(false));
-      logger.debug("Redis command returned: {}",
-          Command.getHexEncodedString(buf.array(), buf.readableBytes()));
+      if (cause == null) {
+        logger.debug("Redis command returned: {} - {}",
+            Command.getHexEncodedString(buf.array(), buf.readableBytes()), extraMessage);
+      } else {
+        logger.warn("Redis command FAILED to return: {} - {}",
```

Comment:
Does this warning message make sense? It might but I thought it would just say it FAILED I'm not sure about "to return"

Reply:
Changed to `debug`